class DataError(Exception):
    """Файл поврежден"""
    pass
